.github/promote-images.yml

```diff
@@ -24,6 +24,7 @@
     workspace-ruby-3: "20.*"
```

Comment:
**(Unrelated to this PR):** Why we have `3` when we haven't even define it here in [this file](https://github.com/gitpod-io/workspace-images/blob/HEAD/dazzle.yaml) 🤔